code-of-conduct.md

```diff
@@ -31,9 +31,10 @@ Harassment includes, but is not limited to:
 - Sustained disruption of discussion
 - Unwelcome sexual attention, including gratuitous or off-topic sexual images or behaviour
 - Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
-- Continued one-on-one communication after requests to cease
+- ** Continued one-on-one communication after requests to cease **
 - Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect others from intentional abuse
 - Publication of non-harassing private communication
+- No gender segragation ( this includes you ladies and gents ) No women or men only channels private or public.  Remember this is a community not a resort.
```

Comment:
s/segragation/segregation

Reply:
Actually I don't know why private channels are even allowed period.  That kind of breaks the no exclusive group rules.

Reply:
>Harassment and other exclusionary behavior aren’t acceptable.

Private channels are not exclusionary behavior, they for information that does not fit well in public channels for a variety of valid reasons. For instance, why are you excluding me from your banking information? Why are you excluding me from the group of people that have keys to your home? These are ridiculous exclusion claims similar to arguing we should not have private channels. Maybe we should get rid of direct messages too. Why not just close down the slack as we are excluding everyone that can not afford a computer or an internet connection.

Reply:
lol, I don't think personal data or DM fall into the "exclusive" category.  Private channels create private exclusive groups of people.  They create places that take away from the community as a whole.